### ISSUE TITLE
JsonProperties without value generate empty keys in json.

### ISSUE THREAD

**Issue by sumupdvs** [organics2016/pojo2json#16]
If you have a Class e.g.  :+1: 

```
public class TransferAccount {
    @JsonProperty
    private String type;

    @JsonProperty("institution_name")
    private String institutionName;
}
```

It will generate the following json:

``` 
"transfer_account": {
          "": "",
          "institution_name": ""}
```

Is it possible to fallback to the field name as declared in the java class when jsonproperty is present but no value is defined.

**Comment by sumupdvs** [comment]
https://github.com/organics2016/pojo2json/blob/master/src/main/java/ink/organics/pojo2json/POJO2JsonAction.java#L178
`
if (StringUtils.isNotBlank(fieldName)) {
                return fieldName;
            } else {
return field.getName();
}
`

**Comment by organics2016** [comment]
> Is it possible to fallback to the field name as declared in the java class when jsonproperty is present but no value is defined.

Thanks for your issue.
I can fix this bug in the next version.

**Comment by organics2016** [comment]
Fixed in release 1.1.7
